Tests/test_neuron_module.py

```diff
@@ -65,7 +65,7 @@ def test_update_cache_var(self):
         expected_dict = {
             "cache": True
         }
-        self.assertEquals(NeuronModule._update_cache_var(True, args_dict=args_dict),
+        self.assertEquals(NeuronModule._update_params({'cache': True}, args_dict=args_dict),
```

Comment:
ok, you've fixed the test. but now you should also add a test with the new parameter: the tts.